src/backend/distributed/executor/multi_executor.c

```diff
@@ -66,7 +66,8 @@ static CustomExecMethods RouterSingleModifyCustomExecMethods = {
 	.ExplainCustomScan = CitusExplainScan
 };
 
-static CustomExecMethods RouterMultiModifyCustomExecMethods = {
+/* not static to enable swapping in multi-modify logic during router execution */
+CustomExecMethods RouterMultiModifyCustomExecMethods = {
```

Comment:
This seems like a bad idea. What if there's other queries in-progress that refer to `RouterMultiModifyCustomExecMethods`? With cursors there could be many custom scan nodes referencing `RouterMultiModifyCustomExecMethods`.